XCUITests/TopTabsTest.swift

```diff
@@ -0,0 +1,162 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCTest
+
+let url = "www.mozilla.org"
+let urlLabel = "Internet for people, not profit — Mozilla"
+let urlValue = "mozilla.org"
+
+let urlYah = "www.yahoo.com"
+let urlLabelYah = "Yahoo"
+let urlValueYah = "yahoo"
+
+let urlExample = "example.com"
+
+class TopTabsTest: BaseTestCase {
+    var navigator: Navigator!
+    var app: XCUIApplication!
+
+    override func setUp() {
+        super.setUp()
+        app = XCUIApplication()
+        navigator = createScreenGraph(app).navigator(self)
+    }
+
+    override func tearDown() {
+        super.tearDown()
+    }
+
+    func testAddTabFromSettings() {
+        navigator.goto(NewTabMenu)
+        app.collectionViews.cells["NewTabMenuItem"].tap()
```

Comment:
Could it be due to the next line also goes to the same place?

> scene.tap(app.buttons["Close Menu"], to: NewTabScreen)

Just tried commenting it and then it works...so, not sure if it is better to keep the tap on + to add a tab from NewTabMenu in the test or add it to the BrowserTapMenu because I guess that removing the close action is not an option :)